### ISSUE TITLE
Can "version" field be given higher precedence?

### ISSUE THREAD

**Issue by octogonz** [cameronhunter/prettier-package-json#32]
The  `"name"` and `"version"` are arguably the most important **package.json** fields.  People probably rely on these two fields more than any other when trying to understand **package.json** relationships.

With the current ordering, `"version"` is buried below less interesting fields. These other fields are also optional, which means their shape changes from file to file:

```js
{
  "name": "prettier-package-json",
  "description": "Prettier formatter for package.json files",
  "license": "MIT",
  "author": "Cameron Hunter <hello@cameronhunter.co.uk>",
  "repository": {
    "type": "git",
    "url": "https://github.com/cameronhunter/prettier-package-json"
  },
  "bugs": "https://github.com/cameronhunter/prettier-package-json/issues",
  "version": "2.1.3",  // <----- 🔍 where's the version... oh there it is!
  "main": "src/index.js",
  "bin": {
    "prettier-package-json": "./bin/prettier-package-json"
  },
```

Can we move it higher up?
```js
{
  "name": "prettier-package-json",
  "version": "2.1.3",
  "description": "Prettier formatter for package.json files",
  "license": "MIT",
  "author": "Cameron Hunter <hello@cameronhunter.co.uk>",
  "repository": {
    "type": "git",
    "url": "https://github.com/cameronhunter/prettier-package-json"
  },
  "bugs": "https://github.com/cameronhunter/prettier-package-json/issues",
  "main": "src/index.js",
  "bin": {
    "prettier-package-json": "./bin/prettier-package-json"
  },
```

The change to [keyOrder](https://github.com/cameronhunter/prettier-package-json/blob/HEAD/src/defaultOptions.js) could be like this:

```js
  keyOrder: [
    /**
     * Details
     */
    '$schema',
    'name',
    'version', // <----- moved up
    'private', // <----- moved down, because this determines the meaning of the "version"
    'description',
    'license',
    'author',
    'maintainers',
    'contributors',
    'homepage',
    'repository',
    'bugs',
    'type',
```

(These options are customizable. My interest here is to improve the "opinionated" standard that **prettier-package-json** is promoting.)

**Comment by canac** [comment]
I second this. @cameronhunter, is that a change you'd be open to considering?

**Comment by cameronhunter** [comment]
[Key order is configurable via options](https://github.com/cameronhunter/prettier-package-json#options) if you have your own preferences. But if you'd like to make a pull-request to change the default order then I'm open to this change.

As time has moved on, my preferences have also changed. Currently I like:

```
keyOrder: [
  "$schema",
  "private",
  "name",
  "version",
  "description",
  …
];
```

**Comment by cameronhunter** [comment]
Resolved in v2.3.0
